ADAL/src/request/ADAuthenticationRequest+Broker.m

```diff
@@ -306,14 +307,25 @@ - (void)callBroker:(ADAuthenticationCallback)completionBlock
     id<ADTokenCacheDataSource> dataSource = [_tokenCache dataSource];
     if (dataSource && [dataSource isKindOfClass:[ADKeychainTokenCache class]])
     {
+        NSString* keychainGroup = [(ADKeychainTokenCache*)dataSource sharedGroup];
+        NSString* teamId = [ADKeychainUtil keychainTeamId:&error];
```

Comment:
is it possible for `[ADKeychainUtil keychainTeamId:&error];` to return something like an empty string and also an error?